backend/prisma/schema.prisma

```diff
@@ -74,4 +75,13 @@ model Transactions {
   time                  DateTime @default(now()) // column to store the timestamp at which a trasaction was logged so we can fetch most recent
   @@index([time])
   transaction_date      Int? // this stores a basic int to test over time, entered manually
+}
+
+model Goals {
+  id        Int     @id @default(autoincrement())
+  users     User[] // array of users that have signed up for this goal
+  category  String // the category this goal is for ex. legal_services
+  value     Int // the percent this goal is targeting users to raise/lower their spending positive values represent raising values (only for investment_and_saving), otherwise value is negative
```

Comment:
Would it make sense to name this something like `target` instead?

Reply:
I think that's fair yeah, I'll commit a change for that now

Reply:
changed in 6119427